osscameroon/camerdevs
# Add get salaries endpoint

This pull request add the GET salaries endpoint

Steps to verify:
- move to the `backend` folder with `cd ./backend`
- run `make start-postrges` to run an initialized database
- run  `make run` to launch the api
- then run the command `curl -X GET localhost:7000/salaries` you should see something like this
```
[{"title":"Recruiting Manager","salary":1624669,"city":"Bobigny","company":"Livefish","createdat":"0001-01-01T00:00:00Z","updatedat":"0001-01-01T00:00:00Z"},{"title":"Analog Circuit Design manager","salary":680067,"city":"Meniko","company":"Riffwire","createdat":"0001-01-01T00:00:00Z","updatedat":"0001-01-01T00:00:00Z"},{"title":"Associate Professor","salary":2562805,"city":"Yaohua","company":"Feedfish","createdat":"0001-01-01T00:00:00Z","updatedat":"0001-01-01T00:00:00Z"},{"title":"Senior Cost Accountant","salary"
....
]
```
